docs/runbooks/KubevirtHyperconvergedClusterOperatorNMOInUseAlert.md

```diff
@@ -1,43 +1,59 @@
 # KubevirtHyperconvergedClusterOperatorNMOInUseAlert
+<!--apinnick Nov 2022-->
 
 ## Meaning
-Starting from version 4.11.0, Node Maintenance Operator (NMO) is no longer included in OpenShift Virtualization.  
-This alert is being fired if both of the following conditions are met:
-- OpenShift version is 4.11 or higher.
-- The integrated NMO is being used, i.e. one or more custom resources of Kind `NodeMaintenance` and API group of `nodemaintenance.kubevirt.io` have been found on the cluster.
 
-Existence of such a custom resource indicates that the targeted node specified in `spec.nodeName` was put into a maintenance mode, which means the node is [cordoned](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#cordon) and server-side [drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service).
+This alert fires when integrated Node Maintenance Operator (NMO) custom resources (CRs) are detected on OpenShift Virtualization 4.10.
+
+The Node Maintenance Operator is not included with OpenShift Virtualization 4.11.0 and later.
+
+<!--KVstart-->
+The presence of `NodeMaintenance` CRs belonging to the `nodemaintenance.kubevirt.io` API group indicates that the node specified in `spec.nodeName` was put into maintenance mode. The target node has been [cordoned off](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#cordon) and [drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service).
```

Comment:
Does it make sense to have upstream doc links when only downstream users are going to see this alert?

Reply:
I think that we can keep the links in this case, unless we have the equivalent links also d/s.

Reply:
I can't put links in the d/s docs. If the links point to OpenShift docs, then they are cross-references. We are not allowed to put cross-references in modules.

So the best I can do is to mark the links with comments so that they only appear in u/s.